plugins/ExhibitGalleryPage/ExhibitGalleryPagePlugin.php

```diff
@@ -13,25 +13,29 @@ public function hookInstall() {
         // add gallery page to all current exhibits.
         $sql = "SELECT id FROM `{$db->prefix}exhibits` ORDER BY id";
         $id_for_exhibits = $db->query($sql)->fetchAll();           
-        foreach ($id_for_exhibits as $exhibit_id) {
+        foreach ($id_for_exhibits as $exhibit_id) {         
           $id = $exhibit_id['id'];
-          $sql_order = "select MAX(`order`) FROM `{$db->prefix}exhibit_pages` b WHERE b.exhibit_id = $id";
-          $max_order = $db->fetchOne($sql_order);
-          // make the gallery page the last one in the navigation menue
-          $newPageData = array (
-            'parent_id' => null, 
-            'exhibit_id' => $id,
-            'title' => 'Gallery',
-            'slug' => 'gallery',
-            'order' =>$max_order+1,
-          );
-          $db->getAdapter()->insert($db->exhibit_pages,$newPageData); 
+          $sql_gallery_page = "select title FROM `{$db->prefix}exhibit_pages` b where (b.exhibit_id = $id and slug = 'gallery')";
+          if (!$db->fetchOne($sql_gallery_page)) 
```

Comment:
Do you want an `else { ... }` clause to ensure that the existing gallery page is the last page?